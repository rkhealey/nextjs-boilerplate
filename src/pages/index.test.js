import { shallow } from 'enzyme';
import React from 'react';

import App from './index';

describe('main page', () => {
  it('renders"', () => {
    const app = shallow(<App />);

    expect(app).toMatchSnapshot();
  });
});
