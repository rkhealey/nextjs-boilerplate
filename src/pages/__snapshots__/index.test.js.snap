// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`main page renders with count 1`] = `
<div>
  The current count is 
  10
</div>
`;
